src/viur/core/bones/relational.py

```diff
@@ -401,74 +401,60 @@ def serialize(self, skel: "SkeletonInstance", name: str, parentIndexed: bool) ->
 
         :raises AssertionError: If a programming error is detected.
         """
+
+        def serialize_dest_rel(in_value: dict | None = None):
+            if not in_value:
+                return None, None
+            if dest_val := in_value.get("dest"):
+                ref_data_serialized = dest_val.serialize(parentIndexed=indexed)
+            else:
+                ref_data_serialized = None
+            if rel_data := in_value.get("rel"):
+                using_data_serialized = rel_data.serialize(parentIndexed=indexed)
+            else:
+                using_data_serialized = None
+
+            return using_data_serialized, ref_data_serialized
+
+
         super().serialize(skel, name, parentIndexed)
+
         # Clean old properties from entry (prevent name collision)
-        for k in list(skel.dbEntity.keys()):
+        for k in skel.dbEntity.keys():
```

Comment:
this can be simplified even further:
But can you please use a more meaningful variable than `k`?
```suggestion
        for k in skel.dbEntity:
```

Reply:
> But can you please use a more meaningful variable than `k`?

@phneutral (/cc @ArneGudermann ) Can you please make sure you read all the comments. Just because there's a suggestion in there doesn't mean that it's the only point in the comment.
It is always tedious to find and reopen conversations in which you have commented on something that has not yet been implemented.

Reply:
Sorry for that, didn't recognize it, as GitHub shows the suggestion as outdated.